### ISSUE TITLE
To add feature and make it more functional in side bar

### ISSUE THREAD

**Issue by krish-karmacharya** [brave/brave-browser#34484]
## Description 

I guess I encounter a feature lack in the side bar as the sites opened form the side bar open in the same tab which is inconvenient while we would already be working in the same tab

## Actual result:

For making things convenient, let's suppose I am playing YouTube on one tab and when I try to access another website form side bar ; ChatGpt let's say, it opens upon the YouTube which I was using already and ChatGpt overlaps it. It's inconvenient and it would be great if I could open in new tabs through side bar without altering currently running tabs

## Expected result:
Tabs not being overlapped with each other while opening from the side bar .

**Comment by bsclifton** [comment]
cc: @rebron

**Comment by Emi-Emi-Emi** [comment]
You only need to press `CTRL` to open in a **new tab** and ALT+SHIFT to open in a **new window**, just like you would do with bookmarks.

I guess that's the problem with sidebar in the current state today, any website added to it works 100% exactly like bookmarks.
We should actually question here, **what is the point of sidebar item vs bookmarks?**
Sidebar only makes sense for the items that open in the sidebar, like `Leo`, `Bookmarks`, `Reading list`, `Playlist`, but not for Brave talk or Wallet, or any 'pinned' website.

This current behavior in sidebar makes adding sites to the sidebar redundant and unnecessary feature in a way. Bookmarks are more 'powerful' because you can access them from omnibox, and use sync and if you right click them, you get the options to even open them in a private window and all that.

In the end, the primitive implementation of website Sidebar items is the problem, sidebar websites  **should all open in the sidebar**, like the the _real_ sidebar items like Leo and Playlist do.
This should become a reality with https://github.com/brave/brave-browser/issues/33533 (?), And if I am correct it does, that means Brave sidebar website items will behave completely different to what they do now, so there is no reason to open in the same tab or in a new tab or new window or anything like that.
Of course, if/when opening websites in the side panel gets implemented, maybe there could be a button when you right click a website in the sidebar, get a new menu to _Open website in a new tab_  instead of opening a website just in the sidebar, this behavior should never be a default setting, if not we go back to sidebar being redundant to what bookmarks have done for years.

**Comment by krish-karmacharya** [comment]
@rebron 
@Emi-Emi-Emi 
I completely agree with your perspective.Adding applications like ChatGPT or Spotify to the sidebar can indeed enhance the focus on the current task. Having quick access to these tools without the need to switch tabs or open separate windows can improve workflow efficiency and concentration. It's a great idea to consider expanding the functionality of the sidebar to include more applications that users frequently use during their tasks. This could provide a more seamless and integrated user experience, making it easier to access essential tools without disruptions.Looking forward to the potential changes with #33533 and the enhanced functionality it might bring to the Brave sidebar! By the way, I wanted to mention that this is my first request, so I appreciate your patience and understanding.

**Comment by sangwoo108** [comment]
I believe clicking the shortcut in sidebar with command/control pressed will open the site in a new tab.

**Comment by sangwoo108** [comment]
Oh, looks like duplicated with https://github.com/brave/brave-browser/issues/31816
